test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,46 @@
+const shouldFail = require('../helpers/shouldFail');
+
+const PausableCrowdsale = artifacts.require('PausableCrowdsaleImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('../helpers/setup');
+
+contract.only('PausableCrowdsale', function ([_, pauser, wallet, anyone]) {
```

Comment:
```suggestion
contract('PausableCrowdsale', function ([_, pauser, wallet, anyone]) {
```

Reply:
y u no use suggestion

Reply:
It said I didn't have permission to commit to this branch, probably because it's in @TalAter's fork.